### PR TITLE
Sort docstring and kwargs of Points layer

### PR DESCRIPTION
**Pull request by Czaki**
# References and relevant issues
similar to #6779, required for #6780 

# Description

For better discoverability, sort positional parameters of Points Layer Constructor. 
Respectively, sort Parameters in docstring. 

Required as we plan to add two new parameters in #6780

### CODE REVIEW

File: napari/layers/points/points.py [hunk 1]
```diff
@@ -115,6 +98,19 @@ class Points(Layer):
...
+    border_width : float, array
+        Width of the symbol border in pixels.
+    border_width_is_relative : bool
+        If enabled, border_width is interpreted as a fraction of the point size.
+    cache : bool
+        Whether slices of out-of-core datasets should be cached upon retrieval.
+        Currently, this only applies to dask arrays.
+    canvas_size_limits : tuple of float
+        Lower and upper limits for the size of points in canvas pixels.
+        experimental_clipping_planes : list of dicts, list of ClippingPlane, or ClippingPlaneList
```
Comment (psobolewskiPhD) [inline napari/layers/points/points.py]:
```suggestion
    experimental_clipping_planes : list of dicts, list of ClippingPlane, or ClippingPlaneList
```
This one is a kwarg so it is too deeply intended

**Review by psobolewskiPhD — APPROVED** [review]
If anyone is wondering why the + and - of the diff don't match:
- experimental clipping planes (+4) and projection_mode (+3) are now in the Parameters docstring and an empty line was removed (-1)

**Comment by psobolewskiPhD** [comment]
Caught a small nitpick.
Any reason not to sort the attributes section of the docstring too?

**Comment by Czaki** [comment]
> Any reason not to sort the attributes section of the docstring too?

Minimize diff and simplify review.

**Pull request merged**